src/jit/codegenarm.cpp

```diff
@@ -1414,6 +1465,18 @@ instruction CodeGen::genGetInsForOper(genTreeOps oper, var_types type)
         case GT_ROR:
             ins = INS_ror;
             break;
+        case GT_ADD_LO:
+            ins = INS_add;
```

Comment:
Handling of this has been added to genCodeForBinary(), because INS_adds is one of encoding of INS_add with flags. (same for INS_subs and INS_sub).